centos.sh

```diff
@@ -358,6 +352,8 @@ write_grub() {
 run_os_specific_functions() {
 
   execute_chroot_command "chkconfig iptables off"
+  execute_chroot_command "chkconfig ip6tables off"
+  execute_chroot_command "chkconfig postfix off"
```

Comment:
why do we disable postfix? The default config just allows sending cron mails, which is normally fine and enabled by most other ISPs.

Reply:
Taken from Hetzner upstream, we should double check that distribution shipped default configurations of postfix are not an open relay before leaving postfix enabled IMHO.